include/swift/SILOptimizer/Analysis/ArraySemantic.h

```diff
@@ -131,6 +134,10 @@ class ArraySemanticsCall {
   /// Returns true on success, false otherwise.
   bool replaceByValue(SILValue V);
 
+  /// Replace a call to append(contentsOf: ) with a series of
+  /// append(element: ) calls.
+  bool replaceByAppendingValues(SILModule &M, SILFunction* appendFn, llvm::SmallVectorImpl<SILValue> &Vs, ArrayRef<ProtocolConformanceRef> &conformances);
```

Comment:
Wrapped the lines at 80 characters. Not sure what you mean by the * and &, but I did move the * so it's against the name rather than the type.